packages/flutter/test/painting/text_painter_test.dart

```diff
@@ -1348,16 +1345,14 @@ void main() {
       PlaceholderDimensions(size: Size(50, 30), alignment: ui.PlaceholderAlignment.bottom),
     ]);
 
-    Object? e;
-    try {
-      painter.paint(MockCanvas(), Offset.zero);
-    } catch (exception) {
-      e = exception;
-    }
     // In tests, paint() will throw an UnimplementedError due to missing drawParagraph method.
     expect(
-      e.toString(),
-      isNot(contains('TextPainter.paint called when text geometry was not yet calculated')),
+      () => painter.paint(MockCanvas(), Offset.zero),
```

Comment:
nit: I guess here the original test code is trying to verify that it doesn't throw that particular error, so the new condition is a bit too strong?

Reply:
Righto. Let's try with `isNot()`.